### PR TITLE
75 support legacy endpoints

### PR DESCRIPTION
**Pull request by erinreeves**
#75

### CODE REVIEW

File: api/src/integrationTest/groovy/ncei/catalog/LegacyEndpointSpec.groovy [hunk 1]
```diff
@@ -18,41 +20,114 @@ class LegacyEndpointSpec extends Specification {
...
 
-  def postBody = [
-      trackingId  : 'ABCD',
-      filename    : 'myfile',
-      dataset     : 'test',
-      fileSize    : 42,
-      geometry    : 'POLYGON((0 0) (0 1) (1 1) (1 0))',
-      fileMetadata: 'this is some raw scraped metadata from a header or whatever'
-  ]
+  def 'HAPPY PATH: metadata-recorder POST gets expected response and etl is able to search for results via paging'() {
```
Comment (zebdelk) [inline api/src/integrationTest/groovy/ncei/catalog/LegacyEndpointSpec.groovy]: Had a verbal conversation about adding cleanup so we can test saving and paging results as separate tests.

Comment (zebdelk) [inline api/src/integrationTest/groovy/ncei/catalog/LegacyEndpointSpec.groovy]: Concluded that at this level, there's a lot of work to setup this kind of cleanup, and little need for it. Ideally our E2E type tests should stay small and limited to ensure wiring doesn't break. No change needed.

**Comment by erinreeves** [comment]
test

**Comment by erinreeves** [comment]
close #73 close #75

**Pull request merged**